### PR TITLE
Update medical items

### PR DESCRIPTION
**Pull request by kil0byt3**
riflemen +1 TQ, +1 PK Blister (2 uses of pk)
medics + naloxone, +bottle of painkillers (2 for SM, 4 for PM)

arty + plotting board

### CODE REVIEW

**Comment by kil0byt3** [comment]
I did this on my phone and it was not easy this app sucks

**Comment by AChesheireCat** [comment]
When and why did we turn off the difference in bandages? It's even baked into some of our medical documentation. Should we turn that back on?

**Comment by kil0byt3** [comment]
ready for review with changes from https://discord.com/channels/204621032428929025/1293996766106222612

**Comment by kil0byt3** [comment]
riflemen +1 TQ,
medics + naloxone, +bottle of painkillers (2 for SM, 4 for PM)
ftls +1 bottle of painkillers for their FT(thoughts?)

**Comment by lambdatiger** [comment]
> ftls +1 bottle of painkillers for their FT(thoughts?)
Maybe a separate PR for that? Maybe we can make a custom painkiller magazine/item that is like a 1-2 count? Not 100% sure.

**Comment by AChesheireCat** [comment]
I concur with Lambda - I think a preferable options would be if we had "painkiller blister packs" that held like 1-2 or 3-4 uses. If we make a 1-2 option, we'd even be incentivized to just issue those to everyone as well :)

For now, I think it's fine if just the Medics receive painkiller bottles imo

**Comment by kil0byt3** [comment]
No more FTL PK, do it the other way in separate PR (but ideally would get merged at the same time as this one)

File: Loadouts/common.hpp [hunk 1]
```diff
@@ -6,9 +6,9 @@
 #define RADIO_LR "ACRE_PRC117F"
 
 // GEAR
-#define BASE_MEDICAL "ACE_elasticBandage:4","ACE_packingBandage:2","ACE_tourniquet","ACE_splint"
-#define MEDIC_MEDICAL RADIO_MR,"ACE_elasticBandage:25","ACE_packingBandage:15","ACE_epinephrine:10","ACE_salineIV:2","ACE_salineIV_500:4","ACE_salineIV_250:8","ACE_morphine:16","ACE_tourniquet:6","ACE_splint:12","ACE_adenosine:10"
-#define PL_MEDIC_MEDICAL RADIO_MR,"ACE_elasticBandage:15","ACE_packingBandage:10","ACE_epinephrine:10","ACE_salineIV:8","ACE_salineIV_500:8","ACE_salineIV_250:8","ACE_morphine:5","ACE_tourniquet:3","ACE_splint:6","ACE_adenosine:10"
+#define BASE_MEDICAL "ACE_elasticBandage:4","ACE_packingBandage:2","ACE_tourniquet:2","ACE_splint"
+#define MEDIC_MEDICAL RADIO_MR,"ACE_elasticBandage:25","ACE_packingBandage:15","ACE_epinephrine:10","ACE_salineIV:2","ACE_salineIV_500:4","ACE_salineIV_250:8","ACE_morphine:16","ACE_tourniquet:6","ACE_splint:12","ACE_adenosine:10","potato_naloxone:3","ACE_painkillers:2"
```
Comment (TheCandianVendingMachine) [inline Loadouts/common.hpp:10]: may want to set the base higher

**Review by TheCandianVendingMachine — APPROVED** [review]

File: Loadouts/common.hpp (hunk 1 above)
Comment (kil0byt3) [reply Loadouts/common.hpp:10]: What?

Comment (TheCandianVendingMachine) [reply Loadouts/common.hpp:10]: oh lmao for painkillers i meant. 2 painkiller items => 10 pills for medics. one pill per person in a reduced squad, not enough for a full squad

Comment (kil0byt3) [reply Loadouts/common.hpp:10]: gotcha. I didn't want to go too crazy with amount of pills if each person is going to have their own 2 (or, originally, if an FTL also had their own bottle of 10)

i'll see if the other idea gets implemented and bump this up if it doesn't

Comment (kil0byt3) [reply Loadouts/common.hpp:10]: other idea was implemented, added pkblister to each rifleman instead of upping the medic's base loadout amt

**Comment by kil0byt3** [comment]
ready for review with new PK blister pack implemented (https://github.com/BourbonWarfare/POTATO/pull/583)

**Review by AChesheireCat — APPROVED** [review]

**Review by lambdatiger — APPROVED** [review]
LGTM

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**